fvtest/jitbuildertest/Makefile

```diff
@@ -39,7 +39,8 @@ OBJECTS := \
 	CallReturnTest \
 	FieldNameTest \
 	ConvertBitsTest \
-	UnsignedDivRemTest
+	UnsignedDivRemTest \
+  SelectTest
```

Comment:
indendation difference

Reply:
Strangely this portion of the file was using tabs for indentation and 2 spaces throughout the file. I'll fix that.